http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -1225,16 +1225,30 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
 
       // Build delete namespace request
       jobId = UUID.randomUUID()
-      deleteNamespaceRequest = new DeleteControlledAzureResourceRequest().jobControl(
+      deleteResourceRequest = new DeleteControlledAzureResourceRequest().jobControl(
         new JobControl().id(jobId.toString)
       )
 
-      _ <- logger.info(ctx.loggingCtx)(s"WSM delete namespace request: ${deleteNamespaceRequest}")
+      _ <- logger.info(ctx.loggingCtx)(s"WSM delete resource request: ${deleteResourceRequest}")
 
       // Execute WSM call
-      result <- F.blocking(
-        wsmApi.deleteAzureKubernetesNamespace(deleteNamespaceRequest, workspaceId.value, wsmResource.resourceId.value)
-      )
+      result <- wsmResource.resourceType match {
+        case WsmResourceType.AzureKubernetesNamespace =>
+          F.blocking(
+            wsmApi.deleteAzureKubernetesNamespace(deleteResourceRequest,
+                                                  workspaceId.value,
+                                                  wsmResource.resourceId.value
+            )
+          )
+        case WsmResourceType.AzureDatabase =>
+          F.blocking(
+            wsmApi.deleteAzureDatabaseAsync(deleteResourceRequest, workspaceId.value, wsmResource.resourceId.value)
+          )
+        case _ =>
+          F.raiseError(
+            AppDeletionException(s"Unexpected WSM resource type for deletion polling ${wsmResource.resourceType}")
+          )
```

Comment:
I am curious the likelihood we are going to hit this as this is a new error condition

Reply:
we really shouldn't hit this error case because of the case in `deleteWsmResource` where this is called, but the compiler yells at me if I don't include an 'else' case....